pkg/cmd/openshift-sdn/cmd.go

```diff
@@ -116,7 +116,7 @@ func (sdn *OpenShiftSDN) Run(c *cobra.Command, errout io.Writer, stopCh chan str
 
 	<-stopCh
 	time.Sleep(500 * time.Millisecond) // gracefully shut down
-	os.Exit(1)
+	os.Exit(0)
```

Comment:
just remove this, I think by default if you return nil the exit code is 0

Reply:
good point; fixed.